### DISCUSSION TITLE
How to keep some property of EntitySpec unserialized?

### DISCUSSION THREAD

**Discussion by Dines97** [dotnet/dotnet-operator-sdk#549] [Q&A]
Hi,

I want to add embedded resource property to my entity spec, but instead of getting serialized class I want to serialize using some custom code in my controller, so I can choose serialization target by myself. What class I should use for this property? JObject, JRaw, object?

After this I still want to serialize object but choose target according to some other criteria. So basically, it still needs to be serializable. What would be appropriate type for this?

`[EmbeddedResource] public <Type> Parameters { get; set; }`

Thanks for everything.

**Comment by buehler** [comment]
Hey @Dines97 
You can try a combination of `[Ignore]` and some other types. The embedded resource is used to serialize other entities (whole entities) in the spec. In theory, you could just use a binary field (base64) or a string field and serialize your entity in there. I just don't know if there are any limits in terms of field size.
